### ISSUE TITLE
add ability to show errors inline like VS Code(error lens)

### ISSUE THREAD

**Issue by victororlyk** [zed-industries/community#1219]
### Check for existing issues

- [X] Completed

### Describe the feature

It would be really nice to have ability to see the error inline without need to hover or go with cursor to that place. 
I really love this extension, but I think it would be nice to have in the code of editor

**Comment by fdionisi** [comment]
@victororlyk is not precisely the same as the error lens plugin, but have you already tried pressing `f8` (and/or `shift+f8`)? It brings you to the next diagnostic and inlines the error.

**Comment by victororlyk** [comment]
@fdionisi  yeah i have, those are cool, but I don't use f keys a lot. I have keyboard with f row and it is a bit annoying to use fs i use them only when it is really needed) https://happyhackingkb.com/. So a for me i just use vim keybinding to jump to the error for me it is faster)

**Comment by restfuladi** [comment]
Agree. This is key feature still keeping me on JetBrains (with Inspection Lens plugin).

**Comment by probablykasper** [comment]
For context, error lens looks like this, with red/orange line and text:
![image](https://github.com/zed-industries/community/assets/11315492/ddcb5b72-c544-4932-86ad-c04a74ac38e1)
